diff --git a/amavisd b/amavisd
index 278f0dd..97e3204 100755
--- a/amavisd
+++ b/amavisd
@@ -26474,7 +26474,8 @@ BEGIN {
   import Amavis::rfc2821_2822_Tools;
   import Amavis::Util qw(ll do_log do_log_safe min max minmax add_entropy
                          untaint untaint_inplace format_time_interval
-                         truncate_utf_8 orcpt_encode idn_to_ascii
+                         truncate_utf_8 orcpt_encode
+                         idn_to_utf8 idn_to_ascii mail_addr_idn_to_ascii
                          safe_encode safe_encode_utf8 safe_decode_mime
                          snmp_count ccat_split ccat_maj);
   import Amavis::Lookup qw(lookup lookup2);
