diff -up amavisd-agent.org amavisd-agent
--- amavisd-agent.org	2008-07-14 13:24:20.000000000 -0400
+++ amavisd-agent	2008-07-14 13:25:00.000000000 -0400
@@ -146,7 +146,7 @@ sub fmt_ticks($) {
       { die "Usage: $0 [ -w <wait-interval> ]\n" }
   }
   my($env) = BerkeleyDB::Env->new(
-    '-Home'=>'/var/amavis/db', '-Flags'=> DB_INIT_CDB | DB_INIT_MPOOL);
+    '-Home'=>'/var/spool/amavisd/db', '-Flags'=> DB_INIT_CDB | DB_INIT_MPOOL);
   defined $env or die "BDB no env: $BerkeleyDB::Error $!";
   my($db) = BerkeleyDB::Hash->new(
     '-Filename'=>'snmp.db', '-Flags'=>DB_RDONLY, '-Env'=>$env );
diff -up amavisd-nanny.org amavisd-nanny
--- amavisd-nanny.org	2008-07-14 13:23:54.000000000 -0400
+++ amavisd-nanny	2008-07-14 13:24:13.000000000 -0400
@@ -56,7 +56,7 @@ my($idlettl) = 3*60*60; # idle children 
 my($activettl) = 10*60; # stuck active children are sent a SIGTERM
                         #   after this many seconds
 
-my($db_home) = '/var/amavis/db';  # DB databases directory
+my($db_home) = '/var/spool/amavisd/db';  # DB databases directory
 my($dbfile)  = 'nanny.db';
 my($wakeuptime) = 2;  # sleep time in seconds, may be fractional
 
